### DISCUSSION TITLE
Could it be added a new  features in the future: The built-in file explorer can show all drives by default on Windows：

### DISCUSSION THREAD

**Discussion by redomCL** [tomasklaen/uosc#952] [Ideas]
Could it be added a new  features in the future: The built-in file explorer can show all drives by default on Windows10/11：
![357273636-07e3c056-9eb0-49e7-891c-9c83ad8d8d7d](https://github.com/user-attachments/assets/8b0ddda0-db3c-4738-a05c-0a7e71eb32d8)

**Comment by tomasklaen** [comment]
Since https://github.com/tomasklaen/uosc/commit/61c259b1377230662c12143f8e23eefdf6fc9348, you can do:

```
default_directory={drives}
```
